fv/infrastructure/topology.go

```diff
@@ -243,18 +259,23 @@ func StartNNodeTopology(n int, opts TopologyOptions, infra DatastoreInfra) (feli
 			if i == j {
 				continue
 			}
-
-			jBlock := fmt.Sprintf("10.65.%d.0/24", j)
-			if opts.IPIPEnabled && opts.IPIPRoutesEnabled {
-				err := iFelix.ExecMayFail("ip", "route", "add", jBlock, "via", jFelix.IP, "dev", "tunl0", "onlink")
-				Expect(err).ToNot(HaveOccurred())
-			} else if opts.VXLANMode == api.VXLANModeNever {
-				// If VXLAN is enabled, Felix will program these routes itself.
-				err := iFelix.ExecMayFail("ip", "route", "add", jBlock, "via", jFelix.IP, "dev", "eth0")
-				Expect(err).ToNot(HaveOccurred())
-			}
+			wg.Add(1)
+			go func(i, j int, iFelix, jFelix *Felix) {
+				defer wg.Done()
+				defer ginkgo.GinkgoRecover()
+				jBlock := fmt.Sprintf("10.65.%d.0/24", j)
+				if opts.IPIPEnabled && opts.IPIPRoutesEnabled {
+					err := iFelix.ExecMayFail("ip", "route", "add", jBlock, "via", jFelix.IP, "dev", "tunl0", "onlink")
+					Expect(err).ToNot(HaveOccurred())
+				} else if opts.VXLANMode == api.VXLANModeNever {
+					// If VXLAN is enabled, Felix will program these routes itself.
+					err := iFelix.ExecMayFail("ip", "route", "add", jBlock, "via", jFelix.IP, "dev", "eth0")
+					Expect(err).ToNot(HaveOccurred())
+				}
+			}(i, j, iFelix, jFelix)
```

Comment:
I'm surprised if there is a lot of time saving here.  Have you measured this one?

Reply:
Yes, it was a few hundred millis IIRC; seemed worth it given we have 1000+ tests.